source/tutorials/nvidia.rst

```diff
@@ -83,11 +83,12 @@ and sustainable on |CL|.
    <https://download.nvidia.com/XFree86/Linux-x86_64/440.44/README/dma_issues.html>`_
    for more information.
 
-   The *intel_iommu-igfx_off* kernel parameter can be removed with the command
-   below: 
+   The *intel_iommu-igfx_off* kernel parameter can be removed with the
+   commands below: 
   
    .. code-block:: bash
     
+      sudo mkdir -p /etc/kernel/cmdline-removal.d/
```

Comment:
@puneetse Does this also require the clr-boot-manager update?
`sudo clr-boot-manager update`

It's similar to [what you recommend here](https://community.clearlinux.org/t/how-to-remove-quiet-kernel-option/693).

Reply:
I'm pushing this through in the interest of users who may need it. 
Please follow up on my comment regardless of merge status.